selfify/sidebar/sidebar.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="sidebar.css"/>
+    
+    <style>
+    
+    </style>
+    
+  </head>
+
+<body>
+  <input type="file" accept="image/*" id="input">
+  
+  <p></p>
+  
+  <div id="drop_zone">
+    <strong>Drag an image file into this Drop Zone ...</strong>
```

Comment:
Again it would be good to give this `<strong>` a bit of breathing room, for instance:

```
#drop_zone_label {
  margin: 1em;
  display: block;
}
```